data/phi1348/abo011/phi1348.abo011.perseus-eng2.xml

```diff
@@ -4,9 +4,9 @@
 	<teiHeader>
 		<fileDesc>
 			<titleStmt>
-				<title><rs key="Julius Caesar">Julius Caesar</rs><rs key="Forum of Caesar"/><rs key="Temple of Julius Caesar"/> from The Lives of the Caesars</title>
```

Comment:
Yes, I see what you mean. the `<rs key=>` looks to be the decision of the editor of this batch. I wouldn't keep these, especially in the text titles.